apps/wistia-videos/src/components/ConfigScreen.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ConfigScreen from './ConfigScreen';
+import { render, screen, fireEvent, configure } from '@testing-library/react';
+
+// configure({testIdAttribute: 'data-test-id'})
```

Comment:
Whats happening here? Everything is commented out

Reply:
Good question. This is just copied over from the existing app. We can add a task to the "improvements" story we have open to address these tests. (For now I'd consider it out of scope to make other improvements to the current code. My goal was just minimal changes to get things working.)